client/my-sites/site-settings/index.js

```diff
@@ -57,12 +57,21 @@ export default function() {
 	);
 
 	if ( config.isEnabled( 'manage/site-settings/disconnect-flow' ) ) {
+		page( '/settings/disconnect-site/:step(confirm)?', mySitesController.sites );
```

Comment:
Is this to cover the case that shows a placeholder forever? Is `:step(confirm)?` necessary, or could it just be `confirm?`.

Not a big deal, just curious 🙂

Reply:
#18749 will add other possible `steps` :slightly_smiling_face: